src/components/Capture/capture.js

```diff
@@ -252,12 +182,7 @@ class Capture extends Component {
     this.setError('SERVER_ERROR')
   }
 
-  onGlareWarning = () => {
-    this.setWarning('GLARE_DETECTED')
-  }
-
   setError = (name) => this.setState({error: {name, type: 'error'}})
```

Comment:
I've fixed the bug and changed the `setError` function within capture to only return errors.